static/app/views/explore/savedQueries/savedQueriesTable.tsx

```diff
@@ -43,22 +51,42 @@ const ORDER: Array<GridColumnOrder<keyof SavedQuery | 'options' | 'access'>> = [
 type Column = GridColumnHeader<keyof SavedQuery | 'options' | 'access'>;
 
 type Props = {
-  mode: 'owned' | 'shared';
+  cursorKey?: string;
+  mode?: 'owned' | 'shared' | 'all';
   perPage?: number;
+  searchQuery?: string;
+  sort?: SortOption;
 };
 
-export function SavedQueriesTable({mode, perPage}: Props) {
+export function SavedQueriesTable({
+  mode = 'all',
+  perPage,
+  cursorKey = 'cursor',
```

Comment:
we need this because the all queries page has two tables that both have their own pagination